.github/workflows/ci.yml

```diff
@@ -59,7 +59,7 @@ jobs:
           node_version: ${{ matrix.node_version }}
 
       - name: E2E tests in chrome
-        run: npm run e2e:chrome && npm run e2e:vue-showcase:chrome
+        run: npx nx affected --target=run e2e:chrome && npx nx affected --target=run e2e:vue-showcase:chrome
```

Comment:
@ahmedsilat44 thank you very much for your changes, but the goal of the PR is to _not_ run every e2e test on every CI run.

For example, if a vue framework related package changes, only run the vue-showcase e2e tests, because there's no change in the showcase app, which is written in angular. Nx deals with that with its affected e2e command

We would like to run `npm run affected:e2e -- --browser=chrome --headless` in CI.
Please change those accordingly in the ci.yml file to get this PR merged.

Reply:
Sorry for the confusion.

You don't need to do anything specific, there is a script set up in the `package.json` called `affected:e2e` which already handles checking when to run which e2e tests.

So the only thing you need to do, is to update the ci.yml file to run the command:
`npm run affected:e2e -- --browser=chrome --headless`

But change the `--browser=chrome` part to edge, when the tests should run in edge, and to firefox when they should run in firefox according to the yml file.